runtime/Python3/src/antlr4/_pygrun.py

```diff
@@ -165,7 +164,3 @@ def process(input_stream, class_lexer, class_parser):
             process(input_stream, class_lexer, class_parser)
         else:
             print("[ERROR] file {} not exist".format(os.path.normpath(file_name)))
-
```

Comment:
Why remove this ?

Reply:
In general it is bad practice to run a module directly like this (e.g. via `python antlr4/_pygrun.py`). For example, if you do this on a file that has relative imports, you will get errors along the lines of `ImportError: attempted relative import with no known parent package
`

`pygrun` is exposed as a console script in the `pyproject.toml`, meaning that if you install the runtime, you have an executable on the path called `pygrun` (including for editable installs).